src/core/webidl.js

```diff
@@ -131,10 +132,17 @@ function createIdlAnchor(escaped, data, parent) {
   if (isDefaultJSON) {
     return hyperHTML`<a data-lt="default toJSON operation">${escaped}</a>`;
   }
+  const unlinkedAnchor = hyperHTML`<a
+    data-link-type="idl"
+    data-idl="${data.partial ? "partial" : null}"
+    data-title="${escaped}"
```

Comment:
Maybe unescaped name?

```suggestion
    data-title="${data.name}"
```